src/algebra/direct_sum.lean

```diff
@@ -15,20 +25,22 @@ variables (ι : Type v) [decidable_eq ι] (β : ι → Type w) [Π i, add_comm_g
 
 def direct_sum : Type* := Π₀ i, β i
 
+notation `⨁` binders `, ` r:(scoped f, direct_sum _ f) := r
+
 namespace direct_sum
```

Comment:
This should do the job, right?
```suggestion
namespace direct_sum

localized notation `⨁` binders `, ` r:(scoped f, direct_sum _ f) := r
```

Reply:
This doesn't look like the right syntax. It should be `localized notation `⨁` binders `, ` r:(scoped f, direct_sum _ f) := r in direct_sum`.

Reply:
Does it still need to be within the namespace?

Reply:
Oh. I got it wrong too. `notation ... := ...` needs to be in double-quotes: ``localized "notation `⨁` binders `, ` r:(scoped f, direct_sum _ f) := r" in direct_sum``.

Reply:
And it doesn't have to be within the namespace.